src/components/implementation/no_interface/scheddev/fprr/Makefile

```diff
@@ -0,0 +1,9 @@
+#C_OBJS=unit_schedlib.o
```

Comment:
Yes, I can do that. I was thinking that it would be time consuming and will first push my current changes. I can undo the directory changes for now if this is not acceptable.